docs/content/development/build.md

```diff
@@ -5,7 +5,7 @@ layout: doc_page
 ### Build from Source
 
 You can build Druid directly from source. Please note that these instructions are for building the latest stable of Druid. 
-For building the latest code in master, follow the instructions [here](https://github.com/druid-io/druid/blob/master/docs/content/development/build.md).
+For building the latest code in master, follow the instructions [here](https://github.com/druid-io/druid/blob/master/docs/content/development/build.html).
```

Comment:
I think this was intended to link to build.md since it instructs how to build the latest code in the repository.